src/phantom.js

```diff
@@ -70,6 +78,14 @@ export default class Phantom {
     }
 
     /**
+     * Registers a callback to receive {String} messages from the Phantom context.
+     * @param {Function}
+     */
+    onMessage(cb) {
```

Comment:
Right, this is very much a hack, as I indicate in the PR title. Yes, it solves my current problem. But at the same time, this appears to be a way to send arbitrary messages back into Node. So I thought I'd bring it to your attention, in case you have advice on how I could make it fit better with the library. If we could implement a `sendMessage()` function, the rest does not seem so out of line with the library (e.g. you use `>` to prefix command responses, I use a random string I generated). Perhaps the `command` interface could be extended to support "response only" "commands" (or... "messages")?